tests/storage/datavolume.go

```diff
@@ -261,47 +259,44 @@ var _ = SIGDescribe("DataVolume Integration", func() {
 			)
 			dataVolume, err := virtClient.CdiClient().CdiV1beta1().DataVolumes(testsuite.GetTestNamespace(nil)).Create(context.Background(), dataVolume, metav1.CreateOptions{})
 			Expect(err).ToNot(HaveOccurred())
-
-			vmi := libstorage.RenderVMIWithDataVolume(dataVolume.Name, dataVolume.Namespace, libvmi.WithCloudInitNoCloud(libvmifact.WithDummyCloudForFastBoot()))
-			vmi = libvmops.RunVMIAndExpectLaunch(vmi, 500)
-
-			By("Expecting the VirtualMachineInstance console")
-			Expect(console.LoginToCirros(vmi)).To(Succeed())
-
-			pvc, err := virtClient.CoreV1().PersistentVolumeClaims(testsuite.GetTestNamespace(nil)).Get(context.Background(), dataVolume.Name, metav1.GetOptions{})
+			libstorage.EventuallyDV(dataVolume, 100, HaveSucceeded())
+			pvc, err := virtClient.CoreV1().PersistentVolumeClaims(dataVolume.Namespace).Get(context.Background(), dataVolume.Name, metav1.GetOptions{})
 			Expect(err).ToNot(HaveOccurred())
 
-			pv, err := virtClient.CoreV1().PersistentVolumes().Get(context.Background(), pvc.Spec.VolumeName, metav1.GetOptions{})
+			executorPod := createExecutorPodWithPVC("size-detection", pvc)
+			fstatOutput, err := exec.ExecuteCommandOnPod(
```

Comment:
Dont you need `Expect(err).ToNot(HaveOccurred())` after executing the command?

Reply:
Right, I missed it

Reply:
Normally there's a linter for that (shoutout @EduardGomezEscandell) but not in kubevirt/kubevirt